### PR TITLE
Clarify SRTP signalling

### LINKED ISSUE #15: SRTP+DTLS description confusing
The description of how DTLS is used for establishing SRTP connections is confusing: is mutual auth in DTLS ever used, or is an OOB fingerprint via SIP always used? The text here needs to be clarified.

### PR DESCRIPTION
**Pull request by csperkins**
This attempts to make the description of DTLS+SRTP keying less confusing.

Closes #15

### CODE REVIEW

File: draft-ietf-taps-transport-security.md [hunk 1]
```diff
@@ -495,27 +493,38 @@ the RTP SSRC (synchronization source), packet index, and session "salting key".
...
+agreement for SRTP {{RFC5763}}.
+Peers use DTLS to perform mutual certificate-based authentication on the
+media path, and to generate the SRTP master key.
+Peer certificates can be issued and signed by a certificate authority.
+Alternatively, certificates used in the DTLS exchange can be self-signed.
+If they are self-signed, certificate fingerprints are included in the signalling
+exchange (e.g., in SIP or WebRTC), and used to bind the DTLS key exchange in
+the media plane to the signaling plane.
+The combination of a mutually authenticated DTLS key exchange on the media
+path and a fingerprint sent in the signalling channel will protect against
```
Comment (chris-wood) [inline draft-ietf-taps-transport-security.md]: s/will protect/protects

File: draft-ietf-taps-transport-security.md [hunk 2]
```diff
@@ -495,27 +493,38 @@ the RTP SSRC (synchronization source), packet index, and session "salting key".
...
+active attacks on the media, provided the signalling can be trusted.
+The signalling needs to be protected as described in, for example, SIP
```
Comment (chris-wood) [inline draft-ietf-taps-transport-security.md]: s/The signalling/Signalling

File: draft-ietf-taps-transport-security.md [hunk 3]
```diff
@@ -495,27 +493,38 @@ the RTP SSRC (synchronization source), packet index, and session "salting key".
...
+- Out-of-order packet receipt.
+- Mutually authenticated key exchange {{RFC5764}}
 
 ### Protocol dependencies
 
-- External key derivation and management mechanism or protocol, e.g., DTLS {{RFC5763}}.
-- External signaling protocol to manage RTP parameters and locate and identify peers, e.g., 
-SIP {{RFC3261}} or WebRTC {{I-D.ietf-rtcweb-security-arch}}.
+- External key derivation and management protocol, e.g., DTLS {{RFC5763}}.
+- External identify management protocol, e.g., SIP Authenticated Identify Management
```
Comment (chris-wood) [inline draft-ietf-taps-transport-security.md]: s/identify/identity, and same for "Identify"

File: draft-ietf-taps-transport-security.md [hunk 4]
```diff
@@ -495,27 +493,38 @@ the RTP SSRC (synchronization source), packet index, and session "salting key".
...
+  {{RFC4474}}, WebRTC Security Architecture {{I-D.ietf-rtcweb-security-arch}}.
```
Comment (chris-wood) [inline draft-ietf-taps-transport-security.md]: Why is this not part of the previous bullet?

**Review by chris-wood — CHANGES_REQUESTED** [review]

File: draft-ietf-taps-transport-security.md (hunk 3 above)
Comment (csperkins) [reply draft-ietf-taps-transport-security.md]: Spell checkers are rubbish :)

File: draft-ietf-taps-transport-security.md (hunk 4 above)
Comment (csperkins) [reply draft-ietf-taps-transport-security.md]: They're separable concerns. One does the key derivation. The other ensures you're doing the key derivation with the right person.

Comment (chris-wood) [reply draft-ietf-taps-transport-security.md]: Ahh, I misread, sorry. I thought these were three separate bullets, not one parent bullet with two children. Carry on.

**Review by chris-wood — APPROVED** [review]

**Pull request merged**